.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,22 +1,19 @@
 ---
-name: Bug Report (.NET Framework)
+name: Bug Report
 about: Create a report to help us improve
 labels: bug
 ---
------------------Bug Template--------------------------------------------------------------
 
-* SDK version: 
-* .NET runtime version (.NET TargetFramework in the .csproj file):
-* Hosting Info (IIS/Azure WebApps/etc): 
-* Platform and OS version: 
+* List of [NuGet packages](https://github.com/microsoft/ApplicationInsights-dotnet#nuget-packages) and version that you are using:
+* Runtime version (e.g. net461, net48, netcoreapp2.1, netcoreapp3.1):
```

Comment:
Would it help to say where to find this info - its in the .csproj under <targetframework> ?